sycl/test-e2e/DeviceLib/cmath_fp64_test.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: aspect-fp64
-// UNSUPPORTED: gpu
+// UNSUPPORTED: hip || cuda
```

Comment:
```
Additionally, this PR changes some of DeviceLib tests were changed from 
being unsupported on GPU to just be unsupported on HIP or CUDA.
```

Is it related to the fix? If yes, we need to update the PR title and description, if not, why we enable these tests on Linux - is there some patch which fix them on Linux?

Reply:
You can ignore my comment.